src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+abstract class Figure implements FigureDraw {

```

Comment:
Don't use abstract classes to set behavior for classes. Consider using an interface to define the behavior if it does not include any implementation details.